meta/saisanitycheck.c

```diff
@@ -449,6 +449,20 @@ bool sai_metadata_is_acl_field_or_action(
         }
     }
 
+    if (metadata->objecttype == SAI_OBJECT_TYPE_UDF_MATCH)
+    {
+        if (metadata->attrid <= SAI_UDF_MATCH_ATTR_GRE_TYPE)
+        {
+            return true;
+        }
+
+        if (metadata->attrid >= SAI_UDF_MATCH_ATTR_L4_DST_PORT_TYPE &&
+                metadata->attrid <= SAI_UDF_MATCH_ATTR_L4_DST_PORT_TYPE)
```

Comment:
why this is range ? you are using the same attribute here SAI_UDF_MATCH_ATTR_L4_DST_PORT_TYPE , use metadata->attrid == SAI_UDF_MATCH_ATTR_L4_DST_PORT_TYPE

Reply:
ACK.